tox.ini

```diff
@@ -1,13 +1,18 @@
 [tox]
 envlist =
+    format
+    lint
     type
     py39
-    lint
 
 [testenv]
 commands =
     pytest {posargs:--cov=reconcile --cov-report=term-missing --cov-report xml}
-deps = -r{toxinidir}/requirements-test.txt
+deps = -r{toxinidir}/requirements/requirements-test.txt
+
+[testenv:format]
+commands = black --check reconcile tools e2e_tests
```

Comment:
Can this be `black {posargs:--check reconcile tools e2e_tests}` which would maintain the default settings, but allow people to override as desired?

Reply:
TIL: `{posargs}`

Thank you!